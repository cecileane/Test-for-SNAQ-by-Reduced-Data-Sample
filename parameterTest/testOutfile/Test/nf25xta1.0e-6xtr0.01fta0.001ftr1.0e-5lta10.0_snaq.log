optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 25, liktolAbs = 10.0.
rootname for files: nf25xta1.0e-6xtr0.01fta0.001ftr1.0e-5lta10.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 52989
seed: 52989 for run 1
Sat Oct  1 17:20:29 2016

 BEGIN SNaQ for run 1, seed 52989 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 75 (failures: 25) with -loglik=128.22643 and ht_min=[0.30771,8.87903,0.4599,1.73905,0.66107]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 4		 3		 1	 |	 NA 		 NA 		 NA 
mvorigin	 17		 17		 1	 |	 0		 0		 --
mvtarget	 14		 14		 1	 |	 0		 0		 --
chdir		 9		 9		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 31		 20		 2	 |	 0		 0		 --
Total		 75		 63		 5	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 128.22642795073662
(2,(4,(3,(5,(6)#H9:::0.6922889010435502):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9:::0.30771109895644977):8.879030533898288);
---------------------
seed: 1943 for run 2
Sat Oct  1 17:20:35 2016

 BEGIN SNaQ for run 2, seed 1943 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 30 (failures: 25) with -loglik=326.11349 and ht_min=[0.49577,0.50581,0.00165,9.00917,0.0,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 4		 4		 0	 |	 0		 0		 --
mvtarget	 12		 12		 0	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 6		 6		 0	 |	 9		 9		 --
Total		 30		 29		 2	 |	 9		 9		 6
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.7

 FINISHED SNaQ for run 2, -loglik of best 326.11348620943636
(2,(3,(5,#H7:0.0::0.49577051738469813):0.5058071172218946):9.009171394638045,(((6,1):0.001652228735913477,4):0.0)#H7:0.0::0.5042294826153019);
---------------------

Sat Oct  1 17:20:38 2016
MaxNet is (2,(4,(3,(5,(6)#H9:::0.6922889010435502):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9:::0.30771109895644977):8.879030533898288); 
with -loglik 128.22642795073662
