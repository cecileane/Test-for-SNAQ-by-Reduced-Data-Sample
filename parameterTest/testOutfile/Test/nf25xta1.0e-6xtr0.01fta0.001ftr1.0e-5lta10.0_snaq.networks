(2,(4,(3,(5,(6)#H9:::0.6922889010435502):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9:::0.30771109895644977):8.879030533898288);, with -loglik 128.22642795073662 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(2,(4,(3,(5,(#H9:::0.34462299987337797,6):1.3813733372997539):0.4616449400903389):0.5168078852703063):2.4729137641870236,(1)#H9:::0.655377000126622);, with -loglik 129.40977520257772
(2,#H9:::0.3554805416251494,(1,((((4)#H9:::0.6445194583748506,3):0.0,5):0.347591868267991,6):1.1434421133306307):7.203638339418084);, with -loglik 176.93266717199796
((4,#H9:::0.42185526077423896):1.943661246427622,3,(5,((1,(2)#H9:::0.578144739225761):4.431748600434059,6):0.15858698209172592):0.03143703940289626);, with -loglik 192.0690780083691
(2,(4,(3)#H9:::0.5622386177881797):0.10356351117081825,(1,((#H9:::0.4377613822118203,5):1.7740100308659101,6):1.0273540877057585):0.09804322925556155);, with -loglik 214.31314225315285
(2,(4,(3,(5)#H9:::0.5022506055714193):1.2009348277254053):0.19512871241146285,(1,(#H9:::0.49774939442858074,6):6.073911272311515):0.050212986879850596);, with -loglik 231.13014230357086
