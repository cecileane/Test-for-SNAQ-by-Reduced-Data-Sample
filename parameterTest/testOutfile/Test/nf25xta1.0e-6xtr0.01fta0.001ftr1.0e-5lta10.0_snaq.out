(2,(4,(3,(5,(6)#H9:::0.6922889010435502):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9:::0.30771109895644977):8.879030533898288); -Ploglik = 128.22642795073662
 Dendroscope: (2,(4,(3,(5,(6)#H9):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9):8.879030533898288);
 Elapsed time: 9.060392243 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (2,(4,(3,(5,(6)#H9:::0.6922889010435502):1.7390532340985179):0.45989802962754994):0.6610709158862911,(1,#H9:::0.30771109895644977):8.879030533898288);, with -loglik 128.22642795073662
 (2,(3,(5,#H7:0.0::0.49577051738469813):0.5058071172218946):9.009171394638045,(((6,1):0.001652228735913477,4):0.0)#H7:0.0::0.5042294826153019);, with -loglik 326.11348620943636
-------
